.github/workflows/pre-commit.yml

```diff
@@ -44,6 +44,7 @@ jobs:
       env:
         # https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/plugins.md#avoiding-rate-limiting
         GITHUB_TOKEN: ${{ github.token }}
+    - run: go install github.com/terraform-docs/terraform-docs@latest
```

Comment:
I think you probably want a section before this that looks like:

```yaml
    - uses: actions/setup-go@v5
      with:
        go-version: '1.23'
        check-latest: true
    - run: go install github.com/terraform-docs/terraform-docs@latest
```

See: https://github.com/actions/setup-go?tab=readme-ov-file#basic